utils/semodule-utils/Makefile

```diff
@@ -0,0 +1,48 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=semodule-utils
+PKG_VERSION:=3.1
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/SELinuxProject/selinux/releases/download/20200710
+PKG_HASH:=0cc37f9cec751d9c2abb5f2b228b060567e973cb47c19b53b8a4a7378baaa853
+PKG_INSTALL:=1
+HOST_BUILD_DEPENDS:=libsemanage/host
```

Comment:
missing `PKG_LICENSE`, `PKG_LICENSE_FILES` and `PKG_CPE_ID` (if assigned)

Reply:
Fixed.